src/runtime/tests/description-test.ts

```diff
@@ -418,23 +417,23 @@ recipe
       assert.isTrue(recipe.isResolved());
 
       const arc = createTestArc(recipe, manifest);
-      const fooStore1 = await arc.createStore(fooType.collectionOf(), undefined, 'test:1') as CollectionStorageProvider;
-      const fooStore2 = await arc.createStore(fooType.collectionOf(), undefined, 'test:2') as CollectionStorageProvider;
+      const fooStore1 = await collectionHandleForTest(arc, await arc.createStore(fooType.collectionOf(), undefined, 'test:1'));
+      const fooStore2 = await collectionHandleForTest(arc, await arc.createStore(fooType.collectionOf(), undefined, 'test:2'));
 
       let description = await test.verifySuggestion({arc}, 'Write to X-foo and write to X-foo.');
       assert.strictEqual(description.getHandleDescription(recipe.handles[0]), 'X-foo');
       assert.strictEqual(description.getHandleDescription(recipe.handles[1]), 'X-foo');
 
       // Add values to the second handle.
-      await fooStore2.store({id: 1, rawData: {name: 'foo-1', fooValue: 'foo-value-1'}}, ['key1']);
-      await fooStore2.store({id: 2, rawData: {name: 'foo-2', fooValue: 'foo-value-2'}}, ['key2']);
+      await fooStore2.add(new fooStore2.entityClass({name: 'foo-1', fooValue: 'foo-value-1'}));
```

Comment:
why store -> add here and below?

Reply:
CollectionHandleNg.add() is apparently the equivalent of the old Collection.store()

Reply:
Ah, yeah I think I remember talking that through with Filippo.